examples/ex9.cpp

```diff
@@ -169,44 +180,65 @@ int main(int argc, char *argv[])
    }
    mesh.GetBoundingBox(bb_min, bb_max, max(order, 1));
 
-   // 5. Define the discontinuous DG finite element space of the given
+   // 5. Define the continous or discontinuous
+   //    finite element space of the given
    //    polynomial order on the refined mesh.
-   DG_FECollection fec(order, dim);
-   FiniteElementSpace fes(&mesh, &fec);
+   FiniteElementCollection *fec;
+   if(fem_type == cg) {
+     fec = new H1_FECollection(order, dim);
+   }else
+   {
+     fec = new DG_FECollection(order, dim);
+   }
+
+   FiniteElementSpace *fes = new FiniteElementSpace(&mesh, fec);
```

Comment:
Do we need fes to be a pointer?

Reply:
That's a good point, its not needed. I can change that.